packages/dd-trace/src/profiling/profilers/cpu.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+class NativeCpuProfiler {
+  constructor (options = {}) {
+    this.type = 'wall'
+    this._samplingInterval = options.samplingInterval || 10 * 1000
+    this._mapper = undefined
+    this._logger = undefined
+    this._pprof = undefined
+  }
+
+  start ({ mapper, logger } = {}) {
+    this._mapper = mapper
+    this._logger = logger
+
+    try {
+      this._pprof = require('pprof')
+    } catch (err) {
+      if (this._logger) {
+        this._logger.error(err)
```

Comment:
Should the rest of the code continue to run or be skipped?

Reply:
I don't really see any _harm_ in continuing to run there, as the other methods are protected against that being not assigned. I'm fine either way though.